compose/kubernetes/superset-staging.yaml

```diff
@@ -0,0 +1,128 @@
+apiVersion: extensions/v1beta1
+kind: Deployment
+metadata:
+  name: superset-staging
+spec:
+  replicas: 1
+  template:
+    metadata:
+      labels:
+        app: superset-staging
+    spec:
+      containers:
+      - name: cloudsql-proxy
+        image: b.gcr.io/cloudsql-docker/gce-proxy:1.09
+        resources:
+          requests:
+            cpu: 100m
+            memory: 100Mi
+        command: ["/cloud_sql_proxy", "--dir=/cloudsql",
+                  "-instances=soru-176122:us-west1:soru-test=tcp:5432",
+                  "-credential_file=/secrets/cloudsql/credentials.json"]
+        volumeMounts:
+          - name: cloudsql-oauth-credentials
+            mountPath: /secrets/cloudsql
+            readOnly: true
+          - name: ssl-certs
+            mountPath: /etc/ssl/certs
+          - name: cloudsql
+            mountPath: /cloudsql
+      - name: superset
+        image: us.gcr.io/soru-176122/superset-staging:latest
```

Comment:
we should be use a fingerprint for each different build. relying on `latest` tag is not a good long term solution.